packages/thumbprint-react/components/Popover/index.jsx

```diff
@@ -0,0 +1,219 @@
+import React, { useEffect } from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Manager, Reference, Popper } from 'react-popper';
+import onClickOutside from 'react-onclickoutside';
+import startsWith from 'lodash/startsWith';
+
+import * as tokens from '@thumbtack/thumbprint-tokens';
+
+import canUseDOM from '../../utils/can-use-dom';
+
+import { TextButton } from '../Button/index.jsx';
+import { Themed } from '../UIAction/index.jsx';
+import { NavigationCloseTiny } from '../../icons/index.jsx';
+
+import styles from './index.module.scss';
+
+const ESC_KEY = 27;
+
+// Internal component only. Proptypes are defined for the main component `Popover` at the end of the
+// file.
+// eslint-disable-next-line react/prop-types
+const PopoverContent = ({ position, isOpen, children, onCloseClick }) => (
+    <Popper
+        placement={position}
+        modifiers={{
+            offset: { offset: `0, ${tokens.tpSpace2}` },
+            preventOverflow: { boundariesElement: 'window' },
+        }}
+        positionFixed={false}
+    >
+        {({ ref, style, placement, arrowProps }) => (
+            <div
+                ref={ref}
+                className={classNames({
+                    [styles.root]: true,
+                    [styles.open]: isOpen,
+                })}
+                style={style}
+                data-placement={placement}
+            >
+                {children}
+
+                <div className={styles.closeButton}>
+                    <TextButton
+                        accessibilityLabel="Close popover"
+                        iconLeft={<NavigationCloseTiny className={styles.closeButtonIcon} />}
```

Comment:
The click target is too small to be accessible. You can work around this by adding padding in `styles.closeButtonIcon`. For an example, see how `ModalDefault` does it.

Reply:
This one still seems to be an issue.